backend/typescript/graphql/types/userType.ts

```diff
@@ -31,8 +31,11 @@ const userType = gql`
 
   extend type Query {
     userById(id: ID!): UserDTO!
+    userByIdCSV(id: ID!): String!
     userByEmail(email: String!): UserDTO!
+    userByEmailCSV(email: String!): String!
```

Comment:
Sorry again for the extra work and being unclear in the ticket, we only need the `usersCSV` mutation for bulk user requests.